code/game/gamemodes/marker/unitologist.dm

```diff
@@ -44,20 +44,20 @@ GLOBAL_LIST_EMPTY(unitologists_list)
 
 /datum/antagonist/unitologist/proc/give_collaborators(mob/living/our_owner)
 	//our_owner.verbs |= /mob/proc/message_unitologists
-	to_chat(our_owner, "<span class='warning'>The marker has established a psychic link between you and your fellow unitologists.</span>")
+	to_chat(our_owner, "<span class='warning'>The marker has established a psychic link between you and your fellow zealots.</span>")
```

Comment:
Instead of just Zealots, how about Unitologist Zealots?

Reply:
I feel like that sentence sounds better without the full name, I'm not sure about that one